packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -68,7 +68,13 @@ class Send extends Component {
                     initialValues={{ from: accountAddress, gasPrice: 4, ...tx }}
                     onSubmit={this.handleSubmit}
                     validate={this.validateForm}
-                    render={({ handleSubmit, valid, validating, values }) => (
+                    render={({
+                      handleSubmit,
+                      form,
```

Comment:
Is this variable used somewhere?

Reply:
ah that was an artifact from the warning engine. removed it in the next commit.